contrib/backporting/cherry-pick

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+set -e
+if [ $# -ne 1 ]; then
+  echo "Usage: $0 <commit-id>"
+  exit 1
+fi
+CID=$1
+if [[ "`git branch -q -r --contains $CID origin/master 2> /dev/null`" != *"origin/master"* ]]; then
+  echo "Commit $CID not in origin/master!"
```

Comment:
Could we try `origin` then try `upstream`? I tend to use `upstream` to refer to the actual upstream repos.

Reply:
I've updated to select the right remote. Does that work for you?